README.adoc

```diff
@@ -50,6 +50,82 @@ With Maven:
 </dependencies>
 ----
 
+[[pulsar-client-reactive-bom]]
```

Comment:
Inspired by our works in Pulsar proper ([here](https://pulsar.apache.org/docs/3.3.x/client-libraries-java-setup/#pulsar-bom)) but minimized a bit.